src/main/java/com/nowoczesnyjunior/financialapp/expensemodule/mapper/ExpenseMapper.java

```diff
@@ -0,0 +1,34 @@
+package com.nowoczesnyjunior.financialapp.expensemodule.mapper;
+
+import com.nowoczesnyjunior.financialapp.expensemodule.model.Expense;
+import com.nowoczesnyjunior.financialapp.expensemodule.model.ExpenseCategory;
+import com.nowoczesnyjunior.financialapp.openapi.model.CategoryDto;
+import com.nowoczesnyjunior.financialapp.openapi.model.ExpenseDto;
+import org.mapstruct.InheritInverseConfiguration;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.MappingConstants;
+
+import java.util.List;
+
+@Mapper(componentModel = MappingConstants.ComponentModel.SPRING, uses = {CategoryDto.class, ExpenseCategory.class})
+public interface ExpenseMapper {
+
+    @Mapping(target = "amount", source = "amount")
+    @Mapping(target = "date", source = "expenseDate")
+    @Mapping(target = "category",source = "category" )
+    @Mapping(target = "description", source = "description")
+    ExpenseDto expenseToExpenseDto(Expense expense);
```

Comment:
You don't need to add `@Mapping` if target name and source are the same

Reply:
Also, If mapper is already named `ExpenseMapper`, consider naming methods as 'toDto' and 'toModel' and move CategoryMapper separetly
We would avoid having repetitions: `expenseMapper.expenseToExpenseDto(savedExpense);`
`expenseMapper.toDTO(savedExpense);`

Reply:
done i done - faktycznie kod robi się od razu bardziej czytelny :)